src/Promitor.Tests.Unit/Validation/Scraper/Metrics/ResourceTypes/LogAnalyticsDeclarationValidationStepsTests.cs

```diff
@@ -111,5 +112,39 @@ public void LogAnalyticsDeclaration_DeclarationWithQuery_Succeeds()
             // Assert
             PromitorAssert.ValidationIsSuccessful(validationResult);
         }
+
+        [Fact]
+        public void LogAnalyticsDeclaration_CustomCloudWithoutLogAnalyticsEndpoint_Fails()
```

Comment:
Let's add one where we do _not_ add log analytics and not specify endpoint; should pass